gtda/mapper/pipeline.py

```diff
@@ -216,14 +216,13 @@ def make_mapper_pipeline(scaler=None,
 
     memory : None, str or object with the joblib.Memory interface, \
         optional, default: ``None``
-        Used to cache the fitted transformers of the pipeline. By default, no
-        caching is performed. If a string is given, it is the path to the
-        caching directory. Enabling caching triggers a clone of the
-        transformers before fitting. Therefore, the transformer instance
-        given to the pipeline cannot be inspected directly. Use the attribute
-        ``named_steps`` or ``steps`` to inspect estimators within the
-        pipeline. Caching the transformers is advantageous when fitting is
-        time consuming.
+        Used to cache the fitted transformers which make up the pipeline. This
+        is advantageous when fitting of early steps is time consuming and only
```

Comment:
i would write "when the fitting ..."

Reply:
Yep, indeed. Excessive copy/pasting from `scikit-learn` is not a good excuse :).